src/TicketingSystem.BusinessLogic/Services/IPaymentService.cs

```diff
@@ -10,16 +10,39 @@ namespace TicketingSystem.BusinessLogic.Services
 {
     public interface IPaymentService : IService<Payment, PaymentDto>
     {
-        public Task<List<EventSectionSeatsModel>> GetPaymentEventSeats(string paymentId, CancellationToken cancellationToken = default);
+        public List<EventSectionSeatsModel> GetPaymentEventSeats(PaymentDto payment, CancellationToken cancellationToken = default);
```

Comment:
Looks like Cancellation token is not needed here

Reply:
Fixed